portfolio.js

```diff
@@ -0,0 +1,37 @@
+const hambergur = document.querySelector('.allign');
+const contacts = document.querySelector('.contacts');
+const closes = document.querySelector('.close');
+const symbols = document.querySelector('.symbols');
+const time = document.querySelector('.time');
+const links = document.querySelector('.links');
+hambergur.addEventListener('click', () => {
+  hambergur.classList.toggle('close');
+  closes.style.display = 'block';
+  hambergur.style.display = 'none';
+  contacts.style.display = 'flex';
+  contacts.style.marginTop = '-60px';
+  contacts.style.marginLeft = '150px';
+  contacts.style.flexDirection = 'column';
+  contacts.style.textDocaration = 'none';
+  contacts.style.color = 'white';
+  contacts.style.position = 'absolute';
+  contacts.style.fontSize = '12px';
+  symbols.style.display = 'none';
+  time.style.display = 'none';
+  links.style.textDecoration = 'none';
+});
```

Comment:
- [ ] [REQUIRED] Please, try to make your code more DRY (Don't Repeat Yourself) because that will make the code easier to maintain. Instead move all of the js lines into a css class and use the js to only toggle this class.

## Example

### Before

```js
hambergur.addEventListener('click', () => {
  hambergur.classList.toggle('close');
  closes.style.display = 'block';
  hambergur.style.display = 'none';
  contacts.style.display = 'flex';
  contacts.style.marginTop = '-60px';
  contacts.style.marginLeft = '150px';
  contacts.style.flexDirection = 'column';
  contacts.style.textDocaration = 'none';
  contacts.style.color = 'white';
  contacts.style.position = 'absolute';
  contacts.style.fontSize = '12px';
  symbols.style.display = 'none';
  time.style.display = 'none';
  links.style.textDecoration = 'none';
});
```

### After

```css
contacts-display {
  display: flex;
  margin-top: -60px;
  margin-left: 150px;
  flex-direction: column;
  text-docaration: none;
  color: white;
  position: absolute;
  font-size: 12px;
}
```

```js
hambergur.addEventListener('click', () => {
  hambergur.classList.toggle('close');
  closes.style.display = 'block';
  contacts.classList.toggle('close');
});
```